applications/FluidDynamicsApplication/python_scripts/fluid_solver.py

```diff
@@ -197,6 +197,8 @@ def _ExecuteCheckAndPrepare(self):
         prepare_model_part_settings = KratosMultiphysics.Parameters("{}")
         prepare_model_part_settings.AddValue("volume_model_part_name",self.settings["volume_model_part_name"])
         prepare_model_part_settings.AddValue("skin_parts",self.settings["skin_parts"])
+        if (self.settings.Has("assign_neighbour_elements_to_conditions")):
```

Comment:
Provided that the defaults are properly checked, this must be always `True`. Am I wrong?

Reply:
I added this check since there are lots of solvers which use fluid_solver as the base class, and if they don't implement this setting in their defaults then the solver will crash. So, would it be ok if I change the existing solver to add this setting and add a deprecation warning if this setting is not found?

Reply:
See [https://github.com/KratosMultiphysics/Kratos/pull/6119/commits/6a030b86b91b08ca08f08f74c6427b3745780ee6](url)

Reply:
I think that it makes sense to have this in all the fluid solvers defaults. Actually, I think that you already added it to all the solvers that derive from the `FluidSolver`.